File: koma-api/src/main/java/org/koma/api/CompileContext.java
package org.koma.api;


import com.helger.css.ECSSVersion;
import com.helger.css.reader.CSSReader;
import org.eclipse.collections.api.factory.Maps;

import java.util.Map;

public class CompileContext {
  private final Map<String, String> a = Maps.mutable.<String, String>empty().asSynchronized();

  public void test() {
    CSSReader.readFromString("", ECSSVersion.CSS30);
  }
}

File: koma-shared/src/main/kotlin/org/koma/shared/context/CompileContext.kt
package org.koma.shared.context

import com.helger.css.ECSSVersion
import com.helger.css.reader.CSSReader
import org.eclipse.collections.api.factory.Maps

class CompileContext {
  private val a: Map<String, String> = Maps.mutable.empty<String, String>().asSynchronized()

  fun test() {
    CSSReader.readFromString("", ECSSVersion.CSS30)
  }
}
